docs/configfile.adoc

```diff
@@ -43,6 +108,9 @@ NOTE: You can use either a `.yml` or `.yaml` extension for this file.
 |===
 |Field | Description
 
+a| inherit
+a| This is used to define absolute or relative path for configuration file from where you want to overwrite paremeters not defined in current config file.
```

Comment:
you don't overwrite parameters that are not defined - there is nothing to overwrite ;-)
I think that enough is saying:
This is used to define absolute or relative path to parent configuration file